File: app/src/main/java/dev/pitlor/smssync/viewmodels/SyncEmptyStateViewModel.java
package dev.pitlor.smssync.viewmodels;

import android.app.Application;
import android.view.View;

import androidx.annotation.NonNull;

public class SyncEmptyStateViewModel extends BaseViewModel {
    public SyncEmptyStateViewModel(@NonNull Application application) {
        super(application);
    }

    public void importExistingData(View v) {
        // Import existing data from a cloud provider
    }
}

File: app/src/main/java/dev/pitlor/smssync/viewmodels/SyncEmptyStateViewModel.kt
package dev.pitlor.smssync.viewmodels

import android.app.Application
import android.view.View
import androidx.hilt.lifecycle.ViewModelInject

class SyncEmptyStateViewModel @ViewModelInject constructor(application: Application) : BaseViewModel(application) {
    fun importExistingData(@Suppress("UNUSED_PARAMETER") v: View?) {
        // Import existing data from a cloud provider
    }
}